examples/fluent-bit/elastic-cloud/README.md

```diff
@@ -37,4 +37,15 @@ This is optional too; another option is to use regular Elasticsearch credentials
         "tls.verify": "Off"
     }
 },
+```
+
+This is optional; if you would like to use Fargate instead of EC2, just add the following in your task definition:
```

Comment:
do you want to note that the execution role is required too? I know its already in the example task def, but may be its worth calling out?

Or instead, may be just link to our docs? https://docs.aws.amazon.com/AmazonECS/latest/userguide/fargate-task-defs.html

Reply:
Thanks for the suggestion, @PettitWesley! Added recommended changes.